README.md

```diff
@@ -1,10 +1,10 @@
-<img alt="OQ-SRTK - The Seismic Site Response Toolkit" class="right" style="width: 60%" src="https://raw.githubusercontent.com/klunk386/SeismicSiteTool/master/Logo/OQ-SRTK-Logo.png" />
+<img alt="OQ-SRTK - The Seismic Site Response Toolkit of OpenQuake" class="right" style="width: 60%" src="https://raw.githubusercontent.com/klunk386/OQ-SiteResponseToolkit/master/logo/OQ-Logo.png" />
```

Comment:
I would have used a much simpler name for the repository. Something like `oq-srtk` in analogy with the `oq-engine` one. More in general I think we should homogenise all our toolkits following this scheme.

Reply:
Is it possible to change the name of the repo once created?